src/main/sensors/gyro.c

```diff
@@ -98,40 +98,42 @@ typedef struct gyroCalibration_s {
 
 bool firstArmingCalibrationWasStarted = false;
 
-typedef union gyroSoftFilter_u {
-    biquadFilter_t gyroFilterLpfState[XYZ_AXIS_COUNT];
-    pt1Filter_t gyroFilterPt1State[XYZ_AXIS_COUNT];
-#if defined(USE_FIR_FILTER_DENOISE)
-    firFilterDenoise_t gyroDenoiseState[XYZ_AXIS_COUNT];
-#endif
-} gyroSoftLpfFilter_t;
+typedef union gyroLowpassFilter_u {
+    pt1Filter_t pt1FilterState;
+    biquadFilter_t biquadFilterState;
+    biquadFilter_t biquadSectionsFilterState[(GYRO_LPF_ORDER_MAX + 1) / 2];   // each section is of second order
```

Comment:
`biquadFilterSections_t biquadFilterSectionState`

Reply:
This separation was intentional - Butterworth filter code has upper limit  on supported order (Q coefficients are needed), but gyro may reduce maximum order to save memory